tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "expo/tsconfig.base",
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
Revert this

Reply:
Removed and now `tsconfig.json` is the same as before. I guess VScode added this line by itself because I didn't touch this file before.